generators/client/templates/angular/src/main/webapp/app/app.module.ts.ejs

```diff
@@ -16,15 +16,16 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-import './vendor.ts';
```

Comment:
Currently README.md file states that you can add dependencies into vendor.ts: https://github.com/jhipster/generator-jhipster/blob/master/generators/common/templates/README.md.ejs#L171  
I this still working after removing this line?

Reply:
I have re-organized the imports. You can check this below below:

```import { icons } from './vendor';```

Reply:
I tested Leaflet and yes, this is working after this change. For me it was confusing that `import { icons } from './vendor';` adds all libraries support from `vendor.ts` behind the scenes.  
So in my testing all is working.

Reply:
Good to hear that everything is working fine. Regarding inclusion of libraries, `webpack` handles that for you. I think that's how module bundlers work. Imports defined in the vendor module are internal to that module unless they are exported to be imported into other modules.

Reply:
I tried to modify code in different ways and found out that if I define variable `const icons2 = ...` in module `app.module.ts` similar to variable `const icons = ...` in module `vendor.ts` and use variable `icons2` instead of `icons`: `iconLibrary.addIcons(...icons2);` then compiler is dropping whole `vendor.ts` file due to optimization and Bootstrap improvement and Leaflet support is gone.  
So this vendor import change seems a little confusing but without modifying code all is working well.

Reply:
As long as you are importing a symbol or default import and using that imported symbol in your current module, it should work fine. If you don't want to use imported symbol, then, you can always fallback to the global import. I don't think we can support both.

Reply:
I believe it's cleaner and better understandable to add new file for exporting FontAwesome icons  array constant, for example `layouts/font-awesome/font-awesome.constants.ts`, and restore global vendor import.